controller/api/util/api_utils.go

```diff
@@ -256,9 +256,12 @@ func BuildStatSummaryRequest(p StatsSummaryRequestParams) (*pb.StatSummaryReques
 // EdgesRequestParams.
 func BuildEdgesRequest(p EdgesRequestParams) (*pb.EdgesRequest, error) {
 	namespace := p.Namespace
-	if namespace == "" && !p.AllNamespaces {
```

Comment:
Don't we want to keep this?  Otherwise if no namespace is specified, we would default to All Namespapces instead of defaulting to the default namespace.

Reply:
This was never the case because the Namespace field defaults to "default" and therefore was never empty.

I originally started to fix this issue by changing the default Namespace value to be empty, but this ended up being a smaller change and requires the same sort of check here.